CharlieBackend.Api/Startup.cs

```diff
@@ -77,9 +79,14 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddControllers()
                 .AddJsonOptions(options =>
-                    {
+                {
                         options.JsonSerializerOptions.Converters.Add(new TimeSpanConverter());
-                    });
+                })
+                .AddFluentValidation(options =>
+                {
+                        options.ValidatorOptions.CascadeMode = CascadeMode.Stop;
+                        options.RegisterValidatorsFromAssemblyContaining<Startup>();
+                });
```

Comment:
@Hlib-mats 
As stated [in documentation](https://docs.fluentvalidation.net/en/latest/aspnet.html#automatic-registration) here:
![documentation-screenshot](https://user-images.githubusercontent.com/38729881/107797335-84ac3400-6d63-11eb-86f4-32584e8ac615.png)
you are automatically registering the validators as Transient services. 

But since they're not depending on any other Transient or Spoced services, we can make our validation perform better by registering validators as Singleton services.

So I believe we can do this:
```suggestion
                .AddFluentValidation(options =>
                {
                        options.ValidatorOptions.CascadeMode = CascadeMode.Stop;
                        options.RegisterValidatorsFromAssemblyContaining<Startup>(lifetime: ServiceLifetime.Singleton);
                });
```

Please, correct me if I'm wrong.

Reply:
We've decided to keep validators as `Transient` or `Scoped`.